common/edl/time.edl

```diff
@@ -14,8 +14,14 @@
 
 enclave
 {
-    include "openenclave/corelibc/time.h"
-    include "openenclave/internal/syscall/types.h"
+    // Needed for integral type time_t
+    include "openenclave/bits/types.h"
```

Comment:
Oh, I didn't see that, good point. I can remove it.

Reply:
I'm going to leave this because time_t isn't considered a builtin type for EDL so getting time_t for free is just luck and could change with the implementation